unattended_scripts/install_functions/opendistro/kibana.sh

```diff
@@ -87,6 +97,16 @@ configureKibana() {
     
     logger "Kibana installed."
 
+    # Edit window title
+    eval "sed -i 's/null, \"Elastic\"/null, \"Wazuh\"/g' /usr/share/kibana/src/core/server/rendering/views/template.js ${debug}"
+    eval "curl -so ~/custom_welcome.tar.gz https://wazuh-demo.s3-us-west-1.amazonaws.com/custom_welcome_opendistro_docker.tar.gz ${debug}"
+    eval "tar -xf ~/custom_welcome.tar.gz -C ~ ${debug}"
+    eval "cp ~/custom_welcome/wazuh_logo_circle.svg /usr/share/kibana/src/core/server/core_app/assets/ ${debug}"
+    eval "cp ~/custom_welcome/wazuh_wazuh_bg.svg /usr/share/kibana/src/core/server/core_app/assets/ ${debug}"
+    eval "cp -f ~/custom_welcome/template.js.hbs /usr/share/kibana/src/legacy/ui/ui_render/bootstrap/template.js.hbs ${debug}"
+    eval "curl -so ~/customWelcomeKibana.css ${resources}/open-distro/kibana/customWelcomeKibana.css ${debug}"
+    eval "cat ~/customWelcomeKibana.css | tee -a /usr/share/kibana/src/core/server/core_app/assets/legacy_light_theme.css ${debug}"
```

Comment:
Maybe we should create a function, e.g. modifyKibanaLogin() or similar with this code, and call it from configureKibana() and configureKibanaAIO(). 

Also, add a empty line between 101 and 102, as these lines references different things, the line 101 modifies the title of the browser page and lines 102 onwards modify the background and logos, the comment on line 100 can imply that the entire block does that when it is not.

Reply:
Fixed in 2410e85817123d565e41a46c548b0b1f959c461f